.github/workflows/main.yml

```diff
@@ -0,0 +1,43 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-node@v2-beta
+      with:
+        node-version: '14'
+    
+    # Runs a single command using the runners shell
+    - name: Install and Build
+      run: |
+        yarn install
+        yarn test
```

Comment:
I'm not sure it makes sense to run tests on build since the tests don't test the `config.json` file. If a user is only changing `config.json` then they wouldn't get much benefit out of the tests. If a power user decides to make a small change to one of the other files we probably wouldn't want to force them to also update the tests. Do you think there's value in running the tests on build that I'm not seeing?

Reply:
I think maybe I misunderstood what you were asking for then. I added this because I thought you wanted a test step here.